docs/user_guide.md

```diff
@@ -10,4 +10,51 @@ Cyclone provides a powerful workflow engine that serves general workflow purpose
 $ make run_examples SCENE=cicd REGISTRIES=<registry>/<project>
 ```
 
-In addition to YAML format manifest files, you can also create your workflows via APIs provided by Cyclone Server. Refer to [API Docs](./swagger-api-docs.md) for detailed APIs.
\ No newline at end of file
+In addition to YAML format manifest files, you can also create your workflows via APIs provided by Cyclone Server. Refer to [API Docs](./swagger-api-docs.md) for detailed APIs.
+
+## Integration
+
+Cyclone provides an integration center for users to integrate external system information, including sensitive information such as `username`, `password`, `token`, etc,
```

Comment:
External systems are called integrations in Cyclone, such as `GitHub`, `docker registry`, `k8s cluster`.  They can be integrated to Cyclone conveniently. There is an integration center in Cyclone Web, where users can manage integrations. From the view of implementation, one integration is saved as a `Secret` in k8s, and information like URL, credentials are hold by it.

Reply:
If we add a separate chapter to describe integration center, there is no need to describe it in detail here.

Reply:
`Workflow Engine` has been described in the `user_guild.md` too, so I think there is no need to add a separate chapter to describe integration.
@supereagle @cd  WDYT

Reply:
If we should move Functional Modules from design doc to user guide, then there is no need to describe here.